frontend_build/src/parse_bundle_plugin.js

```diff
@@ -72,7 +73,8 @@ var parseBundlePlugin = function(data, base_dir) {
       __kolibriModuleName: JSON.stringify(data.name),
       __events: JSON.stringify(data.events || {}),
       __once: JSON.stringify(data.once || {})
-    })
+    }),
+    new extract$trs(path.relative(base_dir, path.dirname(data.stats_file)), data.name)
```

Comment:
How expensive is the string extraction process? Compilation right now takes a bit of time.